mantle/cmd/ore/aws/delete-image.go

```diff
@@ -57,6 +57,18 @@ func runDeleteImage(cmd *cobra.Command, args []string) {
 	}
 
 	if snapshotID != "" {
+		if snapshotID == "detectFromAMI" {
```

Comment:
maybe this should be 
```suggestion
		if snapshotID == "detectFromAMI" && amiID != "" {
```